modules/account-lib/src/coin/near/transferBuilder.ts

```diff
@@ -7,6 +7,7 @@ import * as NearAPI from 'near-api-js';
 import assert from 'assert';
 import { BuildTransactionError } from '../baseCoin/errors';
 import BN from 'bn.js';
+import BigNumber from 'bignumber.js';
```

Comment:
I think we should use just one of this: BN or bignumber both work with Big Numbers

Reply:
Please use bignumber.js

Reply:
near sdk https://www.npmjs.com/package/near-api-js uses bn.js not bignumber.js all interfaces uses that library